pkg/container/qemu_runner.go

```diff
@@ -774,6 +786,52 @@ func createMicroVM(ctx context.Context, cfg *Config) error {
 	)
 }
 
+// getWorkspaceLicenseFiles returns a list of possible license files from the
+// workspace
+func getWorkspaceLicenseFiles(ctx context.Context, cfg *Config) ([]string, error) {
+	// default to root user, unless a different user is specified
+	user := "root"
+	if cfg.RunAs != "" {
+		user = cfg.RunAs
+	}
+
+	// let's create a string writer so that the SSH command can write
+	// the list of files to it
+	var buf bytes.Buffer
+	bufWriter := bufio.NewWriter(&buf)
+	defer bufWriter.Flush()
+	err := sendSSHCommand(ctx,
+		user,
+		cfg.SSHWorkspaceAddress,
+		cfg,
+		nil,
+		nil,
+		nil,
+		bufWriter,
+		false,
+		[]string{"sh", "-c", "cd /mount/home/build && find . -type f -print"},
```

Comment:
you can just do 

```go
		[]string{"sh", "-c", "find /mount/home/build -type f -print"},
```

Reply:
Without looking at the code further, it actually depends if the code is expecting a filename-only output, or if it can deal with absolute pathnames:

```
$ find /tmp/aaa/ -type f -print
/tmp/aaa/c
/tmp/aaa/b
/tmp/aaa/a
$ cd aaa/ && find . -type f -print
./c
./b
./a
```

Reply:
From what I see it later does `filepath.Base()` so it shouldn't be a problem

Reply:
hm, it doesn't seem to work as expected with the proposed change! The workspace ends up with directory structures like these:

```
mount/home/build/LICENSE
mount/home/build/runtime/pack/dist/opt/editorconfig/LICENSE
...
```

So I think the ugly `cd` should stay!